### PR TITLE
more fine grained ElectionTimeout also takes i into account

### PR DESCRIPTION
**Pull request by domsj**
Looks good on CI

### CODE REVIEW

File: src/paxos/multi_paxos_fsm.ml [hunk 1]
```diff
@@ -664,40 +667,36 @@ let wait_for_accepteds (type s) constants state (event:paxos_event) =
...
+            end
+          else
+            begin
+              Logger.debug_f_ "%s: going to RESEND Accept messages" me >>= fun () ->
+              let needed, already_voted = ballot in
+              let msg = Accept(n,i,v) in
+              let silent_others = List.filter (fun o -> not (List.mem o already_voted)) 
+                constants.others in
+              Lwt_list.iter_s (fun o -> constants.send msg me o) silent_others >>= fun () ->
+              mcast constants msg >>= fun () ->
```
Comment (toolslive) [inline src/paxos/multi_paxos_fsm.ml:695]: you could have taken the opportunity to do this as a side effect in ~sides:[....]

File: src/paxos/multi_paxos_type.ml [hunk 2]
```diff
@@ -97,6 +97,6 @@ type effect =
   | ESend of Mp_msg.MPMessage.t * Messaging.id
   | EAccept of (Value.t * n * i)  
   | EStartLeaseExpiration of (Value.t * n * bool (* is slave *))
-  | EStartElectionTimeout of n
+  | EStartElectionTimeout of n * i
```
Comment (toolslive) [inline src/paxos/multi_paxos_type.ml:100]: (n\* i)

**Comment by toolslive** [comment]
LGTM.

**Comment by domsj** [comment]
Merged into 1.7

**Pull request closed without merging**